File: backend/src/main/java/com/tjh/riskfactor/entity/view/AnswerView.java
package com.tjh.riskfactor.entity.view;

import com.fasterxml.jackson.annotation.JsonFormat;

import java.util.Date;

public interface AnswerView {
    Integer getId();
    String getCreator();
    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
    Date getMtime();
}

File: backend/src/main/kotlin/com/tjh/riskfactor/entity/view/AnswerView.kt
package com.tjh.riskfactor.entity.view

import com.fasterxml.jackson.annotation.JsonFormat

import java.util.Date

interface AnswerView {
    val id: Int
    val creator: String
    @get:JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
    val mtime: Date
}
